signal_proc/1. Intro to Red Pitaya.rst

```diff
@@ -0,0 +1,272 @@
+With the same setup as ‎‎2.5.1, change OUT1 to produce a SQUARE, as shown in Fig. 10.
+
+.. image:: media/image1.11.png
+   :name: Configured for SQUARE output
+   :align: center
+
+Fig. 10: OUT1 Configured for SQUARE output
+
+1. Capture a screen shot of the resulting waveforms. Comment on and visible similarities or differences.
+
+2. (Take Home) Try varying amplitudes/frequencies/phases of both channels and comment on the overall effects each variable does as observed 
+   in the time domain. Capture a screen capture that demonstrates each observable change, and clearly label what change was done between 
+   each channel. For any parameters that do not produce visible changes, comment on why you believe this is so.
+
+   a. Amplitude:
+
+   b. Frequency:
+
+   c. Phase:
+
+Sine and Sawtooth
+~~~~~~~~~~~~~~~~~
+
+With the same setup as ‎‎2.5.1, change OUT1 to produce a SAWU, as shown in Fig. 11.
+
+.. image:: media/image1.12.png
+   :name: Configured for SAWU output
+   :align: center
+   
+Fig. 11: OUT1 Configured for SAWU output
+
+1. Capture a screen shot of the resulting waveforms. Comment on any visible similarities or differences.
+
+2. (Take Home) Try varying amplitudes/frequencies/phases of both channels and comment on the overall effects each variable does as observed 
+   in the time domain. Capture a screen capture that demonstrates each observable change, and clearly label what change was done between each channel. 
+   For any parameters that do not produce visible changes, comment on why you believe this is so.
+
+   a. Amplitude:
+
+   b. Frequency:
+
+   c. Phase:
+
+(Take Home) Sine and Pulse Width Modulated (PWM) output
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With the same setup as ‎‎2.5.1, change OUT1 to produce a PWM, as shown in Fig. 12.
+
+.. image:: media/image1.13.png
+   :name: Configured for SAWU output
+   :align: center
+Fig.  12: OUT 1 configured for PWM output
+
+1. Capture a screen shot of the resulting spectrums/spectrograms. Comment on any visible similarities or differences.
+
+2. (Take Home) Try varying amplitudes/frequencies/phases of both channels and comment on the overall effects each variable does as observed in the 
+   frequency domain. Capture a screen capture that demonstrates each observable change, and clearly label what change was done between each channel.
+   For any parameters that do not produce visible changes, comment on why you believe this is so.
+
+   a. Amplitude:
+
+   b. Frequency:
+
+   c. Phase:
+
+   d. Duty Cycle:
+
+Inferences to be made / Questions
+=================================
+
+1. From the previous sets of measurements what instrument(s) would you use to measure each of the following quantities:
+
+A. Amplitude:
+
+B. Frequency:
+
+C. Phase:
+
+Reference text
+==============
+
+For more in-depth documentation, view the official documentation at:
+
+Oscilloscope:
+https://redpitaya.readthedocs.io/en/latest/appsFeatures/apps-featured/oscSigGen/osc.html

```

Comment:
Internal links:      :ref: `Oscilloscope<oscSigGen>` (the sintax is this: :ref:`Displayed_text<internal_reference_name>`)

External links:     |Wikipedia_dB|

 .. |Wikipedia_dB| raw:: html

    <a href="https://en.wikipedia.org/wiki/Decibel" target="_blank">dB scale</a>